inbm-lib/inbm_lib/ota_util.py

```diff
@@ -0,0 +1,25 @@
+"""
+    Module that contains the method for processing OTA resource
+
+    Copyright (C) 2017-2023 Intel Corporation
+    SPDX-License-Identifier: Apache-2.0
+"""
+
+from .xmlhandler import *
+
+
+def create_ota_resource_list(parsed_head: XmlHandler, resource: Dict) -> Dict[str, Any]:
+    """Creates a list of OTA commands requested under POTA along with the resources and arguments
+    associated with each OTA
+
+    @param parsed_head: Parsed head of the manifest xml
+    @param resource: resource to parse
+    @return Dict: A dict containing all the OTAs to be performed
+    """
+    ota_resource_dict = {}
+    for key in resource.keys():
+        ota_resource = parsed_head.get_children(f'ota/type/pota/{key}')
+        if key == 'fota':
+            ota_resource['holdReboot'] = True
+        ota_resource_dict[key] = ota_resource
+    return ota_resource_dict
```

Comment:
If this is only called by the dispatcher agent as I think it is, then it should be in a separate file in the dispatcher agent.  We only should use the inbm-lib for code that is shared between services.

Reply:
Yes, it is only used by dispatcher agent. I have this file to dispatcher agent folder.

Reply:
@nmgaston Integration tests pass. Please have a review again, thanks.